newsroom/types/agenda.py

```diff
@@ -247,6 +247,8 @@ class AgendaItem(ResourceModel, ModelWithVersions):
     planning_items: Annotated[
         list[AgendaPlanningItem], fields.nested_list(include_in_parent=True), Field(default_factory=list)
     ]
+    planning_ids: Annotated[list[fields.Keyword], Field(default_factory=list)]
+    event_ids: Annotated[list[fields.Keyword], Field(default_factory=list)]
```

Comment:
Where are the `event_ids` populated?

Reply:
in the ticket does not specify the functionality or behavior of the `event_ids` field—such as where or how it should be populated. It only mentions that we need to add two fields to the schema

Reply:
That's my bad, I apologise. Here you would need to store all IDs from the related events onto that event id field